doc/Integration.rst

```diff
@@ -0,0 +1,67 @@
+Integration
+===========
+
+Doctrine integration
+--------------------
+
+Independently from your preferred framework's characteristics, it is also possible to use plain
+Doctrine with the Money VO, using its Embeddables_ feature.
+
+.. _Embeddables: http://doctrine-orm.readthedocs.org/projects/doctrine-orm/en/latest/tutorials/embeddables.html
+
+Since you should not use annotations on the third party's library files, you'll want to use the `YAML mapping`_.
```

Comment:
Sorry, after rereading my comment I relized that I was totally unclear: I meant that XML could also be mentioned, the configuration itself is perfectly fine in YAML.